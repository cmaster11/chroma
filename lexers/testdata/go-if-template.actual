has "Hello" (get .headers "x-my-header")

listFindRegex (get .headers "x-my-header") "Hello"

eq 1 2