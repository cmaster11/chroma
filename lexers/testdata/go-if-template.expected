[
  {"type":"NameOther","value":"has"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\"Hello\""},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"("},
  {"type":"NameOther","value":"get"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameAttribute","value":".headers"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\"x-my-header\""},
  {"type":"Operator","value":")"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"NameOther","value":"listFindRegex"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"("},
  {"type":"NameOther","value":"get"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameAttribute","value":".headers"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\"x-my-header\""},
  {"type":"Operator","value":")"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\"Hello\""},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"Keyword","value":"eq"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"1"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"2"}
]
